processor/Cargo.toml

```diff
@@ -23,6 +23,7 @@ std = ["vm-core/std", "winterfell/std", "log/std"]
 log = "0.4.14"
 vm-core = { package = "miden-core", path = "../core", version = "0.4", default-features = false }
 winterfell = { package = "winter-prover", version = "0.4.2", default-features = false }
+hex = { version = "0.4" }
```

Comment:
We should shield the features as hex will pull std by default. They have `alloc` available that will help us here.

Reply:
changed to:
```
hex = { version = "0.4" , default-features = false , features = ["alloc"]}
```